README.md

```diff
@@ -52,30 +52,34 @@ this will run container with your kafka topic declared topic value , in this cas
-- CONFIGURATION_EXPORTER_WORKER_FS_INTERNAL_OUTPUTS_PATH    select the directory name by full path that the worker will be generate the package to, deafult to '/repository/root/location/outputs', in docker: "/app/outputs"
-- CONFIGURATION_EXPORTER_WORKER_S3_ENDPOINT_URL the url endpoint for the S3 service
-- CONFIGURATION_EXPORTER_WORKER_ACCESS_KEY_ID the access key id ("username") credential for the S3 service
-- CONFIGURATION_EXPORTER_WORKER_SECRET_ACCESS_KEY the secret access key ("password") credential for the S3 service
-- CONFIGURATION_EXPORTER_WORKER_BUCKET the bucket we will be using within the S3 service
\ No newline at end of file
+Make sure to append `CONFIGURATION_EXPORTER_WORKER` before each of the environment variables below:
+
+
+- `KAFKA_HOST_IP`   an array of host ips and ports of kafka brokers *can be multiple*
+- `KAFKA_TOPIC`     the topic's name that the worker will be listening to
+- `KAFKA_GROUP_ID`       the group's name that the consumer will be join to
+- `KAFKA_OFFSET_RESET`   can be 'earliest' or 'latest', will decide whether to start from the beginning of the topic (earliest) or from the end of the topic (latest)
+  `KAFKA_POLL_TIMEOUT_MILLISECONDS`    limits the time from consuming the message to the commit if its not failed **by milliseconds** 
+- `KAFKA_POLL_RECORDS`    limits the number of messages that will be consumed at once by the worker default to 1
+- `LOGGER_LEVEL`    select the level of logs, default to 'INFO'
+- `LOGGER_FILENAME`    select the log's file name
+- `EXPORTSTORAGE_URL`    define the protocol, ip and port of the exported storage service, default to http://127.0.0.1:8080 , change if run on other port or on Docker.
+- `STORAGE_PROVIDER` the storage service we are using, currently S3 or file-system are supported, default to FS
+- `GDAL_MULTITHREAD`    boolean to enable [gdal mutlithread](https://gdal.org/programs/gdalwarp.html#cmdoption-gdalwarp-multi) option 
+- `GDAL_THREAD_COUNT`    [gdal wrap options - NUM_THREADS option](https://gdal.org/api/gdalwarp_cpp.html#_CPPv4N15GDALWarpOptions16papszWarpOptionsE) negative number will be converted to "ALL_CPUS"  
+- `GDAL_OUTPUT_SRS`   set target spatial reference default to 'EPSG:4326'
+- `GDAL_OUTPUT_FORMAT`   select the output format, use the short format name, default to 'GPKG'
+- `FS_EXTERNAL_PHYSICAL_PATH`   select the directory name by full path that wil be mounted to outputs directory **mandatory when using FS storage provider** 
+- `FS_INTERNAL_OUTPUTS_PATH`    select the directory name by full path that the worker will be generate the package to, deafult to '/repository/root/location/outputs', in docker: "/app/outputs"
+- `S3_ENDPOINT_URL` the url endpoint for the S3 service
+- `ACCESS_KEY_ID` the access key id ("username") credential for the S3 service
+- `SECRET_ACCESS_KEY` the secret access key ("password") credential for the S3 service
```

Comment:
please create a related issue and PR in the helm charts repo - https://github.com/MapColonies/raster-helm

Reply:
https://github.com/MapColonies/raster-helm/issues/5